api-reference/_hidden/dxPieChartCommonAnnotationConfig/tooltipTemplate.md

```diff
@@ -5,16 +5,18 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+<!-- %shortDescription% -->
 
 ##### param(annotation): dxPieChartAnnotationConfig | any
-<!-- Description goes here -->
+<!-- %param(annotation)% -->
 
 ##### param(element): dxElement
-<!-- Description goes here -->
+<!-- %param(element)% -->
 
 ##### return: String | Element | jQuery
-<!-- Description goes here -->
+<!-- %return% -->
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- Description goes here -->
+
+<!-- import * from 'api-reference\_hidden\dxChartCommonAnnotationConfig\tooltipTemplate.md' -->
```

Comment:
> images/ChartJS/annotations/pie-chart-annotation-tooltip.png

This image is unused.

Reply:
The reason is the tooltipEnabled option is inherited from the baseWidgetAnnotationConfig. And the image in this option  is added as follows:

![DevExtreme {WidgetName}: Annotation Tooltip](/images/ChartJS/annotations/{widget-name}-annotation-tooltip.png)

Now the image in this option is missing, and this image is fixing it: https://js.devexpress.com/Documentation/20_2/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/#tooltipEnabled